earth2studio/models/px/sfno.py

```diff
@@ -222,29 +227,42 @@ def load_default_package(cls) -> Package:
         """Load prognostic package"""
 
         package = Package(
-            "ngc://models/nvidia/modulus/sfno_73ch_small@0.1.0",
+            # "ngc://models/nvidia/modulus/sfno_73ch_small@0.1.0",
+            "https://portal.nersc.gov/cfs/m4416/earth2mip_prod_registry/sfno_linear_74chq_sc2_layers8_edim620_wstgl2-epoch70_seed102/",
```

Comment:
Don't change the default package. Don't put these packages explicitly in the code.